packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -93,14 +107,20 @@ export const main = async (
 		const cmsType = isString(args.cmsType) ? args.cmsType.toLowerCase() : null;
 
 		if (cmsType) {
-			if (['wp', 'drupal', 'any'].indexOf(cmsType) == -1) {
+			if (
+				['wp', 'wordpress', 'drupal', 'd9', 'd10', 'any'].indexOf(cmsType) == -1
```

Comment:
```suggestion
				['wp', 'wordpress', 'drupal', 'd9', 'd10', 'any'].indexOf(cmsType) === -1
```
Very minor, but we almost always want to use the strict equality operator (triple `=`) vs `==` because strict eq doesn't perform type coercion on the left hand of the operands so it is very slightly faster. With good types up front, you shouldn't need `==` ever.